ckanext/external_storage/tests/test_actions.py

```diff
@@ -0,0 +1,72 @@
+from ckan.tests import factories
+from ckan.tests import helpers as test_helpers
+
+from ckanext.external_storage.tests import get_context
+
+
+class TestBlobSotrageActions():
+    """Test cases for logic actions
+    """
+
+    org_admin = factories.User()
```

Comment:
These class-level objects (org_admin, initial_dataset, etc.) are not good, because they are *static* - this is state that is not cleared between tests. Better to use fixtures, or if we have to, test setup / tear-down functions to create them and clear them after each test.